src/main/java/org/wso2/carbon/module/swiftiso20022/validation/common/ValidationResult.java

```diff
@@ -30,6 +30,10 @@ public ValidationResult() {
         this(true, null, null);
     }
 
+    public ValidationResult(boolean isValid) {
+        this(isValid, null, null);
```

Comment:
Why do we need this constructor

Reply:
It was added to improve readability.
Removed the boolean constructor in commit fdf9fe08dfb8b8c04ee04cb393c773ffe3ab319b and replaced used places with the default constructor with comments for clarification.